src/components/ReactPlayer/CustomBarControls.tsx

```diff
@@ -17,27 +22,65 @@ interface MuteConfigProps {
 export interface CustomBarControlsProps extends ClassNameProps {
     mute?: MuteConfigProps;
     elapsedTimePercent?: number;
+    type?: CustomControlsType;
+    isPaused?: boolean;
+    onPlayClick?: () => void;
+    isStarted?: boolean;
 }
 
 const CustomBarControls = (props: CustomBarControlsProps) => {
-    const {mute, elapsedTimePercent = 0, className} = props;
+    const {
+        mute,
+        elapsedTimePercent = 0,
+        className,
+        type = CustomControlsType.Variant1,
+        isPaused,
+        onPlayClick,
+        isStarted = false,
+    } = props;
 
-    const renderMute = useCallback((elapsedTime: number, muteConfig?: MuteConfigProps) => {
-        if (!muteConfig) {
+    const muteButton = useMemo(() => {
+        if (!mute || type === CustomControlsType.Variant2) {
```

Comment:
we try not use naming like `Variant2`, could we call it related to it's purpose or appearance?

Reply:
Yes. I think one can use namings `with-mute-button` and `with-play-pause-button`